Importer/Interpreter/EntityDataInterpreter.php

```diff
@@ -180,14 +183,22 @@ protected function interpretAssociativeRow(array $columns)
                 throw $exception;
             }
 
-            $fields = $fieldConfiguration->getField();
-            $fieldsToInterpret = is_array($fields) ? $fields : [$fields];
-            foreach ($fieldsToInterpret as $field) {
+            $field = $fieldConfiguration->getField();
+            try {
                 $interpretedRow[$field] = $this->interpretField($fieldConfiguration, $value);
+            } catch (InterpreterException $exception) {
+                $errors[] = $exception;
             }
         }
 
-        return $interpretedRow;
+        switch (count($errors)) {
+            case 0:
+                return $interpretedRow;
+            case 1:
+                throw $errors[0];
+            default:
+                throw new MultipleInterpreterException($errors);
```

Comment:
Do we really need to distinguish between these 2 cases (1 error, more errors)? Does interpretation or import logic changes if there is more than 1 error?
Why not just throw exception with error*s* that were found while interpreting a row, if there were any?
So just:

```
if (count($errors) > 0) {
    throw new InterpreterException($errors);
}

return $interpretedRow;
```

and let UI part decide how to present a row error(s).